Sources/PackageLoading/ModuleMapGenerator.swift

```diff
@@ -180,47 +180,27 @@ public struct ModuleMapGenerator {
     }
 }
 
-extension ModuleMapGenerator.ModuleMapError: FixableError {
-    public var error: String {
+extension ModuleMapGenerator.ModuleMapError: CustomStringConvertible {
+    public var description: String {
         switch self {
         case .unsupportedIncludeLayoutForModule(let (name, problem)):
-            return "could not generate modulemap for target '\(name)', the file layout is not " +
-                "supported: \(problem.error)"
-        }
-    }
-
-    public var fix: String? {
-        switch self {
-        case .unsupportedIncludeLayoutForModule(let (_, problem)):
-            return problem.fix
+            return "target '\(name)' failed modulemap generation; \(problem)"
         }
     }
 }
 
-extension ModuleMapGenerator.ModuleMapError.UnsupportedIncludeLayoutType: FixableError {
-    public var error: String {
+extension ModuleMapGenerator.ModuleMapError.UnsupportedIncludeLayoutType: CustomStringConvertible {
+    public var description: String {
         switch self {
         case .umbrellaHeaderWithAdditionalNonEmptyDirectories(let (umbrella, dirs)):
-            return "an umbrella header is defined at \(umbrella.asString), but the following directories " +
-                "exist: \(dirs.map({ $0.asString }).sorted().joined(separator: ", "))"
+            return "umbrella header defined at '\(umbrella.asString)', but directories exist: " +
+                dirs.map({ $0.asString }).sorted().joined(separator: ", ")
         case .umbrellaHeaderWithAdditionalDirectoriesInIncludeDirectory(let (umbrella, dirs)):
-            return "an umbrella header is defined at \(umbrella.asString), but more than 1 directories " +
-                "exist: \(dirs.map({ $0.asString }).sorted().joined(separator: ", "))"
+            return "umbrella header defined at '\(umbrella.asString)', but more than one directories exist: " +
+                dirs.map({ $0.asString }).sorted().joined(separator: ", ")
         case .umbrellaHeaderWithAdditionalFilesInIncludeDirectory(let (umbrella, files)):
-            return "an umbrella header is defined at \(umbrella.asString), but the following files " +
-                "exist: \(files.map({ $0.asString }).sorted().joined(separator: ", "))"
-        }
-    }
-
-    public var fix: String? {
```

Comment:
I think you should still offer helpful instructions on how to resolve these problems; not just state the problem. I'm worried that beginners in Swift(PM) would otherwise be easily confused.